zbus/src/connection/socket/unix.rs

```diff
@@ -319,8 +319,11 @@ fn get_unix_peer_creds_blocking(fd: RawFd) -> io::Result<ConnectionCredentials>
     {
         use nix::sys::socket::{getsockopt, sockopt::PeerCredentials};
 
-        let fd = fd.as_raw_fd();
-        getsockopt(fd, PeerCredentials)
+        // TODO: get this BorrowedFd directly from get_unix_peer_creds(), but this requires a
+        // 'static lifetime due to the Task.
```

Comment:
We need to actually move everything to use Owned and BorrowFd and I've been working on that (to complete #395) this week.

Reply:
Thanks for doing that!  This will nicely improve over this PR. :)